src/pages/activity.js

```diff
@@ -85,28 +84,26 @@ export default function Activity() {
   }, []);
 
   useEffect(() => {
-    let _transfers = daitransfers.concat(mainnetTransfers)
-      // Filter ongoing private events
-      .filter(t => {
-        // Check if transfer belongs to a private event
-        const privateEvent = privateEvents.find(e => e.id === t.token.event.id)
-        if (privateEvent === undefined) return true;
-
-        // If it does, check if it is an ongoing private event
-        let evEndDate = new Date(privateEvent.end_date.replace(/-/g, ' ')).getTime()
-        let now = new Date().getTime()
-
-        // Show any transfers from finished private events
-        return evEndDate <= now;
-      })
-      // Sort from newer to older
-      .sort((a, b) => {
-        return b.timestamp - a.timestamp
-      })
-      // Only show a few
-      .slice(0, transferLimit)
-    setTransfers(_transfers)
-  }, [daitransfers, mainnetTransfers, privateEvents])
+    const setNewTransfers = async () => {
```

Comment:
@pablojp Thinking of a future refactor this logic probably shouldn't be here. Probably it should be within the logic refactor that I mentioned above.